### ISSUE TITLE
[Foxy] Replace / Remove ros2_control_test_nodes dependency

### ISSUE THREAD

**Issue by fmauch** [UniversalRobots/Universal_Robots_ROS2_Driver#347]
The test scripts depend on ros2_control_test_nodes, which is not released for Foxy (and probably will not, since for galactic+ it is included into ros2_controllers). We could provide our own testing script instead.

**Comment by fmauch** [comment]
@destogl you might have an opinion on this, as you've created the script as far as I can see...

**Comment by destogl** [comment]
@bmagyar can we merge and release test nodes also for `foxy`? Do you see any issues/complexities with that as “release master”?

**Comment by fmauch** [comment]
@bmagyar Any updates on this? I've implemented a [hacky workaround](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/373/commits/a54fc607af475138059c4d8cff5e3cb9150352e4) in #373 which would not technically stop us from doing a release, but I don't feel very great about that...

**Comment by fmauch** [comment]
Tagging this with a wontfix for now, going with the hacky #373.

**Comment by jmoon0714** [comment]
Hello, I'm new here. I am trying to build this for foxy on the UR5. I want to run 
`ros2 launch ur_bringup test_joint_trajectory_controller.launch.py`, but I am failing to do for the fact that the executable 'publisher_joint_trajectory_controller' doesn't exist in `ros2_control_test_nodes`. My understanding is that this is because `ros2_control_test_nodes` doesn't exist yet (Or ever will?). 

Is there a hacky workaround for this?

**Comment by fmauch** [comment]
@jmoon0714 I guess your problem just got fixed in #514
